### DISCUSSION TITLE
Retrieve all mutations done to the DB

### DISCUSSION THREAD

**Discussion by tomaszferens** [prisma/prisma#19041] [Q&A]
I would like to get all mutations that were done to the database so that backend can respond with that data to the frontend.

The code below creates two rows: user and organization.

```
prisma.user.create({
    data: {
        name: 'john',
        organizations: {
            create: {
                name: 'org',
            },
        },
    },
});
```

Is there a way to retrieve that informations from prisma? I would like to get something like this:
```
{
    user: {
        create: [], // list of created users
    },
    organization: {
        create: [], // list of created organizations
    }
}
```

**Comment by ajmnz** [comment]
You can add `select` or `include` statements to get data from your `create` call

```ts
const user = await prisma.user.create({
  data: {
    name: "john",
    organizations: {
      create: {
        name: "org",
      },
    },
  },
  include: {
    organizations: true
  }
});

console.log(user); // User & { organizations: Organization[] } ✅
```

**Reply by ludralph** [reply]
Hey @tomaszferens ,

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
